include/vvdec/vvdec.h.in

```diff
@@ -408,7 +408,9 @@ typedef struct vvdecSeqInfo
 
   void*    reservedPtr_1;              // reserved space for future use
   void*    reservedPtr_2;              // ...
-  int64_t  reserved_1;                 // ...
+  int8_t   numReorderPics;             // max reorder pics across all temporal layers
+  uint32_t maxLatencyIncreasePlus1;    // max latency increase plus 1 across all temporal layers
+  char     reserved_1[3];              // reserved (maintains 8-byte alignment)
```

Comment:
Yes, that looks good in principle, but to maintain ABI compatibility the `uint32_t` field needs to be before the `int8_t`, because otherwise the compiler inserts a padding to align the `uint32_t`.
Furthermore, I would prefer the `numReorderPics` to be `uint8_t`, and the `char`s to be `int8_t` for consistency